src/utils/menu.js

```diff
@@ -1,52 +1,85 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Products',
     sublinks: [
       {
-        name: 'Honey Faucet',
-        link: 'https://faucet.1hive.org/',
-        description: 'Register with BrightId to claim Honey'
+        name: 'Honeypot',
+        link: 'https://1hive.org',
+        description: 'DAO Governance'// - Create and stake on proposals'
       },
       {
-        name: '1hive DAO Interface',
-        link: 'https://1hive.org',
-        description: 'Create Proposals and Stake Honey'
+        name: 'Honey Faucet',
+        link: 'https://faucet.1hive.org/',
+        description: 'Claim Free Honey'// - Register with BrightId to claim Honey'
       },
       {
-        name: 'Honeyswap Interface',
+        name: 'Honeyswap',
         link: 'https://honeyswap.org/',
-        description: 'Swap tokens and supply liquidity'
+        description: 'Decentralized Exchange' //Swap & Supply Liquidity - Swap tokens and supply liquidity. Create new pools'
       },
       {
-        name: 'Honeyswap Analytics',
-        link: 'https://info.honeyswap.org',
-        description: 'Honeyswap analytics and historical data'
+        name: 'Honeycomb',
+        link: 'https://hny.farm/',
+        description: 'Yield Farming'// - Stake Honeyswap LP tokens to claim Honey.'
       }
     ]
   },
   {
-    name: 'Documentation',
+    name: 'External Apps',
     sublinks: [
       {
-        name: '1hive Handbook',
-        link: '/docs/dao',
-        description: 'Learn about 1hive and how to contribute'
+        name: 'BrightID',
+        link: 'https://www.brightid.org/',
+        description: 'Verify for Faucet'
       },
-
       {
-        name: 'Honeyswap Docs',
-        link: '/docs/honeyswap'
+        name: 'MetaMask',
+        link: 'https://metamask.io/',
+        description: 'Crypto wallet'
+      },
+      {
+        name: 'OmniBridge',
+        link: 'https://omni.xdaichain.com/',
+        description: 'Any ERC20 token bridge'
+      },
+      {
+        name: 'Blockscout',
+        link: 'https://blockscout.com/poa/xdai/',
+        description: 'Analyze EVM blockchains'
+      },
+      {
+        name: 'DeFi Pulse',
+        link: 'https://defipulse.com/',
+        description: 'DeFi analytics'// - Added to be listed on DeFi Pulse'
       },
     ]
   },
   {
     name: 'Info',
     sublinks: [
+      {
+        name: '1hive Handbook',
+        link: '/docs/dao',
+        description: 'Learn about 1hive and how to contribute'
+      },
+      {
+        name: 'Honeyswap Docs',
+        link: '/docs/honeyswap',
+        description: 'Technical concepts'
+      },
+      {
+        name: 'Honeyswap Analytics',
+        link: 'https://info.honeyswap.org',
+        description: 'Market Analytics'// - In depth Honeyswap market data.'
```

Comment:
It's Honeyswap Info here as well. Also I'd recommend that we keep the descriptions in the same format. Some are title case (like this one) while others are not.

Reply:
Agree with case structure, I will try to use Sentence case. I don't agree with it being called Honeyswap Info as stated above.

Reply:
same as before, people in crypto is used to the info term for the analytics

Reply:
please look into  this further, I think your mistaking the href with what is actually displayed as the titles on all the pages and in all the menus. I maybe mistaken.

Reply:
ah yes i missunderstood my bad